### DISCUSSION TITLE
Clarification on use of exchange for nested/locally refined grid.

### DISCUSSION THREAD

**Discussion by RyanConway91** [MODFLOW-ORG/modflow6#1255] [Q&A]
I am hoping to get some clarification on what exactly is happening when flow exchange is used in nested model/locally refined grid situation (highly refined local model nested in a larger coarse regional model).

Here is what I think is happening:

- For the regional model, all BCs/properties are unimpacted by exchange with the local. Exchange simply enforces the groundwater flow equations at the cells defined in the exchange. For example, a well specified in the local model will not 'show up' in the regional model (excepting any impacts of the well that propagate to the exchange boundary)
- For the local model, a well simulated in the regional model will not 'show up' in the local model (again excepting impacts that occur at the exchange boundary).

Basically what I am trying to confirm is that I don't need to delete boundary conditions in the regional model that overlay the local model and that properties in the two models do not need to be the same where the models overlay.

**Comment by christianlangevin** [comment]
In the regional model, you will need to deactivate (by setting IDOMAIN=0) the area that is replaced with the locally refined child model.  This means that you will also have to remove the boundary conditions in the regional model, where the child model exists, because MODFLOW 6 does not allow boundary conditions in cells with IDOMAIN = 0, and those boundary conditions should be included in the child model.

FloPy has an LGR utility that can help with this.  There are a couple of MODFLOW 6 examples that show this.  The [nested grid problem with two domains](https://modflow6-examples.readthedocs.io/en/master/_examples/ex-gwf-u1gwfgwf.html) and [LGR2 problem 3](https://modflow6-examples.readthedocs.io/en/master/_examples/ex-gwf-lgr.html).
